diff --git a/includes/drush.inc b/includes/drush.inc
index f17cc9b..cc0b452 100644
--- a/includes/drush.inc
+++ b/includes/drush.inc
@@ -110,7 +110,7 @@ function drush_get_class($class_name, $constructor_args = array(), $variations =
       $variant_class_name = $class_name . implode('', array_slice($variations, 0, $i));
       if (class_exists($variant_class_name)) {
         $reflectionClass = new ReflectionClass($variant_class_name);
-        return $reflectionClass->newInstanceArgs($constructor_args);
+        return !empty($constructor_args) ? $reflectionClass->newInstanceArgs($constructor_args) : $reflectionClass->newInstanceArgs();
       }
     }
   }
diff --git a/includes/output.inc b/includes/output.inc
index 66e609f..0af9a2d 100644
--- a/includes/output.inc
+++ b/includes/output.inc
@@ -253,7 +253,7 @@ function _drush_format_table($rows, $header = FALSE, $widths = array(), $console
   // Add defaults.
   $tbl = new ReflectionClass('Console_Table');
   $console_table_options += array(CONSOLE_TABLE_ALIGN_LEFT , '');
-  $tbl = $tbl->newInstanceArgs($console_table_options);
+  $tbl = !empty($console_table_options) ? $tbl->newInstanceArgs($console_table_options) : $tbl->newInstanceArgs();
 
   $auto_widths = drush_table_column_autowidth($rows, $widths);
 
