diff --git a/manifests/ini.pp b/manifests/ini.pp
index 4854863..b6f3e7e 100644
--- a/manifests/ini.pp
+++ b/manifests/ini.pp
@@ -16,52 +16,20 @@
 #   String. Optional. Default: 'extra.ini'
 #   The configuration filename
 #
-# [*sapi_target*]
-#   String. Optional. Default: 'all'
-#   The target sapi for the configuration file.
-#   Bu default it will try to apply the configuration to both cli and http
-#
 define php::ini (
   $value        = '',
   $template     = 'extra-ini.erb',
   $target       = 'extra.ini',
-  $sapi_target  = 'all',
   $service      = $php::service,
   $config_dir   = $php::config_dir
 ) {
 
   include php
 
-  $http_sapi = $::operatingsystem ? {
-    /(?i:Ubuntu|Debian|Mint|SLES|OpenSuSE)/ => '/apache2/',
-    default                                 => '/',
+  file { "${config_dir}/${target}":
+    ensure  => 'present',
+    content => template("php/${template}"),
+    require => Package['php'],
+    notify  => Service[$service],
   }
-
-  if ($sapi_target == 'all') {
-
-    file { "${config_dir}${http_sapi}conf.d/${target}":
-      ensure  => 'present',
-      content => template("php/${template}"),
-      require => Package['php'],
-      before  => File["${config_dir}/cli/conf.d/${target}"],
-    }
-
-    file { "${config_dir}/cli/conf.d/${target}":
-      ensure  => 'present',
-      content => template("php/${template}"),
-      require => Package['php'],
-      notify  => Service[$service],
-    }
-
-  }else{
-
-    file { "${config_dir}/${sapi_target}/conf.d/${target}":
-      ensure  => 'present',
-      content => template("php/${template}"),
-      require => Package['php'],
-      notify  => Service[$service],
-    }
-
-  }
-
 }
